app/src/main/java/org/gdg/frisbee/android/api/SynchronousCallAdapterFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.gdg.frisbee.android.api;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+import retrofit2.Call;
+import retrofit2.CallAdapter;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import timber.log.Timber;
+
+public class SynchronousCallAdapterFactory extends CallAdapter.Factory {
```

Comment:
This was just to keep the changes to a minimum. I agree with you. And we should refactor the calls. Also the use of `Callback` is not consistent.